lib/mix/lib/mix/tasks/test.ex

```diff
@@ -410,8 +424,30 @@ defmodule Mix.Tasks.Test do
     end
   end
 
+  @manifest_file_name ".ex_unit_failures.elixir"
```

Comment:
Since mix is the one specifying the file name, maybe we should call it `.mix_test_failures`?